src/forms.ts

```diff
@@ -1,6 +1,41 @@
 import { Utils } from "./utils";
 
 export class Forms {
+  /**
+   * Checks if the label has validation and apply
+   * the correct class and styles
+   * @params textarea Textarea to be checked
+   */
+  static validate_field(textarea: HTMLInputElement) {
+    // Checks if textarea is exist
+    if (!textarea) {
+      console.error('No textarea element found');
+      return;
+    }
+
+    let hasLength = textarea.getAttribute('data-length') !== null;
+    let lenAttr = parseInt(textarea.getAttribute('data-length'));
+    let len = textarea.value.length;
+
+    if (len === 0 && textarea.validity.badInput === false && !textarea.required ) {
+      if (textarea.classList.contains('validate')) {
+        textarea.classList.remove('valid');
+        textarea.classList.remove('invalid');
+      }
+    } else {
+      
+      if (textarea.classList.contains('validate')) {
```

Comment:
Yeah, that could work. I couldn't find anything referencing the `validate` attribute other than this part. As I stated, this is the code from previous version of it. Not sure if it could be impacting anything else so far.